cmake/Modules_CUDA_fix/upstream/FindCUDA.cmake

```
@@ -719,7 +719,7 @@ if(CMAKE_CROSSCOMPILING)
   elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "arm")
     # Support for arm cross compilation
     set(CUDA_TOOLKIT_TARGET_NAME "armv7-linux-gnueabihf")
-  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64|arm64)$")
```

I don't think we should worry about CUDA support on ARM Mac
```suggestion
  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
```

---

yes. this should be removed.